lookup_plugins/ldap.py

```diff
@@ -14,8 +14,12 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+# 03/21/2018, Hidde van der Heide
+# - Improved Python 3 support and recent python-ldap
 
 from __future__ import absolute_import
+from buildins import str
```

Comment:
Isn't there a typo here? Also, this will work with Python 3 but breaks compatibility with Python 2.

Reply:
Thanks for the quick reply! I noticed the typo as well after creating the pull request.

I think this is the suggested way of having a Python 2 and 3 string representation. A quick test confirms the code runs with 2.7 but I'm open to suggestions